buildkite/graphql/teams.graphql

```diff
@@ -0,0 +1,15 @@
+query getTeam($slug: ID!) {
+  # maybe change the generated type name with @genqlient(typename: "Team")?
+  team(slug: $slug) {
+    defaultMemberRole
+    description
+    id
+    isDefaultTeam
+    membersCanCreatePipelines
+    name
+    # @genqlient(typename: "GenqlientTeamPrivacy")
```

Comment:
I've done this for now because the generated default clashed with the existing `TeamPrivacy` type already present